pages/change.php

```diff
@@ -198,7 +210,7 @@
 ?>
 
 <div class="result alert alert-<?php echo get_criticity($result) ?>">
-<p><i class="fa fa-fw <?php echo get_fa_class($result) ?>" aria-hidden="true"></i> <?php echo $messages[$result]; ?></p>
+<p><i class="fa fa-fw <?php echo get_fa_class($result) ?>" aria-hidden="true"></i> <?php echo $messages[$result] . " " . $ldap_extended_error ;  ?></p>
```

Comment:
As previously said in comment, the extended error should not be shown to end user, but can be logged for administrators

But if you can parse the extended message, you may assign a different error message (`tooshort`, `minlower`, etc.) instead of the default one (`passworderror`).

Reply:
You are right. It is better to interpret the text returned by OpenLDAP, in order to return a translated text. But I would post the original message if I do not have a match.